### PR TITLE
Add fullscreen terminal support

### PR DESCRIPTION
**Pull request by spadgett**
@jwforres FYI I have fullscreen support working for terminals. Need to test more across browsers.

@csrwng CC

TODO:

- [x] Add expand and collapse icons for entering and exiting fullscreen (`fa-expand` / `fa-compress`)
- [ ] Investigate sending ESC to the terminal since it's trapped by the browser to exit fullscreen

### CODE REVIEW

File: app/scripts/controllers/pod.js [hunk 1]
```diff
@@ -316,6 +317,17 @@ angular.module('openshiftConsole')
           }
         };
 
+        var focusTerminal = function() {
+          $('.terminal').focus();
```
Comment (jwforres) [inline app/scripts/controllers/pod.js]: do we actually have multiple terminals when there are multiple containers (with only one visible)? or do we just maintain the connections?

or does jquery only find the visible ones by default, cant remember

Comment (spadgett) [inline app/scripts/controllers/pod.js]: Good catch. It looks like we keep the terminal once it's been used, so it's possible to have multiple. I'll change it to only focus the visible terminal.

**Comment by spadgett** [comment]
Updated

**Comment by spadgett** [comment]
Rebased

**Comment by jwforres** [comment]
[merge]

**Pull request merged**